tests/counter.test.ts

```diff
@@ -0,0 +1,87 @@
+import { tx } from "@hirosystems/clarinet-sdk";
+import { Cl, UIntCV } from "@stacks/transactions";
+import { describe, expect, it } from "vitest";
```

Comment:
It’s nice that the test-discovery mechanism (and the test-runner) are finally decoupled in this new Clarinet version.

/cc @friedger @setzeus